doc/man/scc-hypervisor-collector.1.md

```diff
@@ -0,0 +1,216 @@
+
+## TLS/SSL CERTIFICATES
+
+The **virtual-host-gatherer(1)** framework doesn't currently
+permit the specification of certs that can be used to validate
+connections to the specified hypervisors.
+
+As such any certs that may be needed to securely connect to the
+specified hypervisors will need to be registered with the system
+certificate stores. See **update-ca-certificates(8)** for details.
+
+# SYSTEMD INTEGRATION
+
+The **scc-hypervisor-collector** package, when installed, automatically
+creates the **scchvc** restricted service account, if it doesn't already
+exist, with a home directory of **/var/lib/scchvc**.
+
+Additionally two **systemd(1)** units are also installed:
+
+* **scc-hypervisor-collector.service(8)** runs the **scc-hypervisor-collector**
+  as the **scchvc** user if a valid configuration has been specified in
+  the **scchvc** user account.
+
+* **scc-hypervisor-collector.timer(8)** triggers the
+  **scc-hypervisor-collector.service(8)** unit to be run on a daily
+  basis by default.
+
+# CONFIGURATION SETTINGS
+
+Configuration settings are specified in YAML format and must contain:
+
+* a list of hypervisor **backends** from which to retrieve relevant
+  details
+* a collection of **credentials** that will be used to upload the
+  collected details to the SUSE Customer Center.
+
+See **scc-hypervisor-collector(5)** for details about the possible
+configuration settings.
+
+## CONFIGURATION MANAGEMENT
+The configuration settings can be specified in a single YAML file,
+or as a set of YAML files whose content will be merged together in
+lexical sort order.
+
+The latter scheme allows the configuration settings to be split up
+into multiple files, e.g. credentials can be specified in one file,
+and the hypervisor backends in one or more files.
+
+When splitting the hypervisor backend details among multiple files,
+the **backends** lists in each file will be merged together to form
+a single combined list; exact duplicates will be ignored but partial
+duplicates will result in errors.
+
+## ACCESS AND OWNERSHIP
+For security reasons only the non-root user that is running the
+**scc-hypervisor-collector** command should be able to access the
+specified configuration files.
+
+## CONFIGURATION VALIDATION
+The **--check** option can be utilised to check if the specified
+configuration settings are valid, or will report any errors that
+it detected.
+
+## SUPPORTED HYPERVISORS
+
+The following hypervisor types are supported:
+
+* VMWare vCenter (type **VMware**)
+* Libvirt (type **Libvirt**)
+
+Each hypervisor type has specific configuration settings that must
+be provided to permit the relevant details to be retrieved; these
+settings are documented in **scc-hypervisor-collector(5)**.
+
+# EXIT CODES
+
+**scc-hypervisor-collector** sets the following exit codes:
+
+* 0:  Run completed successfully, or configuration settings
+      are valid if check mode (**--check**) was specified.
+* 1:  An error occurred.
+
```

Comment:
Should error messages be documented?

Reply:
Not necessarily,

We may add/tweak the error code settings here as well, e.g. error code 1 means a general error occurred, error code 2 indicates that we were able to collect & upload details from a subset of the specified hypervisor backends, but that some may have failed.